### ISSUE TITLE
Training speed

### ISSUE THREAD

**Issue by Stangerine** [Anonymous1563/Bayesian-Enhancement-Model#16]
How long does it take to train on an Nvidia 4090? I see the GPU usage is very low. Is this normal?

![Image](https://github.com/user-attachments/assets/320568b8-d949-4c51-a24b-8f7cba797361)

**Comment by Stangerine** [comment]
Is it possible to train with multiple cards?

**Comment by BinCVER** [comment]
If your terminal keeps printing out training loss, then it is normal.

**Comment by BinCVER** [comment]
> Is it possible to train with multiple cards?

No, I have only tested the model on a single GPU. If you attempt multi-GPU training, you may encounter issues related to hyperparameter settings or potential code bugs.
